src/Main.java

```diff
@@ -1888,13 +1888,13 @@ else if (location.equals("E1R")) {
 				*/
                     && inventory.contains(Items.STONE_FOOT)
                     && (canGP(inventory) 
-						|| (inventory.contains(Items.JUMP_BOOTS) && (canSuperSwim(inventory) || (difficulty >= Difficulty.HARD && canSwim(inventory))));
+			|| (inventory.contains(Items.JUMP_BOOTS) && ((canSuperSwim(inventory) || (difficulty >= Difficulty.HARD && canSwim(inventory)))));
```

Comment:
See earlier conversation about this change - the extra `(` added here should be removed

Reply:
Should be fixed fully now. Screenshot attached in the code block to show the proper linking up of the ( before canGP and the final ) in the following line.

![image](https://github.com/AaronDobbe/wl3-randomizer/assets/31748762/04b963c1-b116-4373-b3a0-568790b5321d)